src/HTTPRequest.cpp

```diff
@@ -0,0 +1,125 @@
+#include "../include/HTTPRequest.hpp"
+#include <sstream>
+#include <string>
+
+HTTPRequest::HTTPRequest() {
```

Comment:
Don't forget to initialise the private member variables with the initialiser list.